apps/web/components/Wrapper.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import * as React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+import { LAYOUT } from '@kiwicom/margarita-utils';
+
+import Layout from './Layout';
+
+type Props = {|
+  +children: React.Node,
+|};
+
+const ChildrenWrapper = styled.div({
+  marginTop: defaultTokens.spaceXSmall,
+  marginLeft: '25%',
```

Comment:
Could we have mobile-first approach so that styles are faster to be parsed on mobile? I mean by that that CSS should apply first for mobile, then media query for tablet and finally media query for desktop.

Reply:
Sure

Reply:
🚀